packages/web-app/src/modules/machine-views/components/NewStartButton.tsx

```diff
@@ -0,0 +1,203 @@
+  },
+  startButtonText: {
+    color: theme.darkBlue,
+    position: 'absolute',
+    top: 1,
+    height: 40,
+    width: buttonWidth,
+    fontFamily: theme.fontGroteskBook25,
+    fontSize: 12,
+    '-webkit-app-region': 'none',
+    cursor: 'pointer',
+    '&:hover': {
+      opacity: 0.7,
+    },
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    textTransform: 'uppercase',
+    flexDirection: 'column',
+  },
+  runningTimeText: {
+    fontSize: 8,
+    textAlign: 'center',
+  },
+  errorNotification: {
+    position: 'absolute',
+    top: 0,
+    right: -12.5,
+    cursor: 'pointer',
+    '&:hover': {
+      opacity: 0.8,
+    },
+    // Triangle generated from http://apps.eky.hk/css-triangle-generator/
+    width: 0,
+    height: 0,
+    borderStyle: 'solid',
+    borderWidth: '19.1px 11px 0 11px',
+    borderColor: `${theme.orange} transparent transparent transparent`,
+  },
+  errorIcon: {
+    position: 'absolute',
+    color: 'white',
+    top: -20,
+    left: -2,
+    fontWeight: 'bold',
+  },
+})
+
+interface Props extends WithStyles<typeof styles> {
+  onClick?: () => void
+  onClickError?: () => void
+  status?: MiningStatus
+  isRunning: boolean
+  runningTime?: number
+  notCompatible: boolean
+}
+
+interface State {
+  isHovering: boolean
+}
+
+class _NewStartButton extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      isHovering: false,
+    }
+  }
+
+  handleStart = () => {
+    const { onClick } = this.props
+    onClick?.()
+  }
+
+  handleErrorClick = () => {
+    const { onClickError } = this.props
+
+    onClickError?.()
+  }
+
+  handleMouseEnter = () => {
+    this.setState({
+      isHovering: true,
+    })
+  }
+
+  handleMouseLeave = () => {
+    this.setState({
+      isHovering: false,
+    })
+  }
+
+  render() {
+    const { notCompatible, isRunning, runningTime, status, classes } = this.props
+    const { isHovering } = this.state
+
+    const showStatus = isRunning && !isHovering
+
+    const tenPercent = status === MiningStatus.Initializing && runningTime && runningTime < 60000
+    const twentyPercent =
+      status === MiningStatus.Initializing && runningTime && runningTime > 60000 && runningTime < 600000
+    const thirtyPercent =
+      status === MiningStatus.Initializing && runningTime && runningTime > 600000 && runningTime < 1200000
+    const fortyPercent = status === MiningStatus.Initializing && runningTime && runningTime > 1200000
+
+    return (
+      <div className={classes.startButtonContainer}>
+        <div
+          className={classnames(classes.startButton, {
+            [classes.startButtonInstalling]: status === MiningStatus.Installing,
+            [classes.startButtonPrepping10]: tenPercent,
+            [classes.startButtonPrepping25]: twentyPercent,
+            [classes.startButtonPrepping60]: thirtyPercent,
+            [classes.startButtonPrepping90]: fortyPercent,
```

Comment:
Can be simplified to something like this:

```suggestion
            [classes.startButtonPrepping10]: status === MiningStatus.Initializing && progress <= 0.1,
            [classes.startButtonPrepping25]: status === MiningStatus.Initializing && progress > 0.1 && progress <= 0.2,
            [classes.startButtonPrepping60]: status === MiningStatus.Initializing && progress > 0.2 && progress <= 0.3,
            [classes.startButtonPrepping90]: status === MiningStatus.Initializing && progress > 0.3,
```